conf/main/grakn.properties

```diff
@@ -16,122 +16,224 @@
+# If the provided number of threads = 0 then the number of processors available to
+# the Java virtual machine at startup time will be used.
+loader.threads=0
 
-storage.cassandra.frame-size-mb = 200
+############################# Logging Configuration #############################
 
-# Titan Caching
-cache.db-cache = true
+# Single directory under which to store log files
+log.dirs=../logs/
+# Specify the log verbosity level. This can be one of:
+#
+# ERROR  (critical errors indicating the application has failed)
+# WARN   (errors that will not affect the overall running of the application)
+# INFO:  (minimally verbose, including Grakn engine lifecycle events)
+# DEBUG: (verbose, non production use, server task lifecycle events)
+# TRACE: (extraordinarily verbose, be brave, including graql query traversal paths,
+#         extra task lifecycle events, kafka consumer offsets, etc)
+log.level=INFO
 
-# Titan Cache Refresh Time in milliseconds
-cache.db-cache-clean-wait = 20
+############################# Titan Graph Configuration #############################
+# See the Titan docs (http://s3.thinkaurelius.com/docs/titan/1.0.0/titan-config-ref.html)
+# for more explanations of these properties.
+# Any titan property can be added to this file and will be forwarded to the
+# Titan Graph library.
 
-# Titan Cache Expiry Time in milliseconds
-cache.db-cache-time = 180000
+# The primary persistence provider used by Titan. Grakn recommends using Cassandra
+# as it is the supported backend.
+storage.backend=cassandra
+# The hostname or comma-separated list of hostnames of storage backend servers
+storage.hostname=127.0.0.1
+# Cassandra thrift frame size in megabytes
+storage.cassandra.frame-size-mb=200
+
+#### Cassandra specific configuration ####
+
+# Whether to enable Titan’s database-level cache, which is shared across all transactions.
+# Enabling this option speeds up traversals by holding hot graph elements in memory,
+# but also increases the likelihood of reading stale data. Disabling it forces each transaction
+# to independently fetch graph elements from storage before reading/writing them.
+cache.db-cache=true
+# How long in milliseconds database-level cache will keep entries after flushing them.
+cache.db-cache-clean-wait=20
+# Default expiration time in milliseconds for entries in the database-level cache.
+# Set to 0 to disable expiration (cache entries live forever or until memory pressure
+# triggers eviction).
+cache.db-cache-time=180000
+# Size of Titan's database cache in proportion to JVM size 0 (small) to 1 (large)
+cache.db-cache-size=0.25
+
+# Titan InputFormat configuration for using hadoop
+# Not clear why these need to be set. See
+# http://stackoverflow.com/questions/38524151/counting-vertices-on-a-titan-graph-using-sparkgraphcomputer-throws-org-apache-sp/38529076
+# for more information.titanmr.ioformat.conf.storage.backend=cassandra
+titanmr.ioformat.conf.storage.hostname=localhost
```

Comment:
These options are set for analytics and are passed directly to spark I believe.
They need to always match storage.hostname so maybe we can do something about the duplication?